main.go

```diff
@@ -110,7 +110,7 @@ func main() {
 		}
 	}
 
-	conf, err := config.LoadConfig(config_path)
+	conf, err := config.LoadConfigFromFile(config_path)
```

Comment:
Naming is good. 🥳

Reply:
Long name is better than ambiguous name. 🎉